contracts/ethereum_contracts.go

```diff
@@ -550,22 +551,23 @@ func (l *EthereumLinkToken) Transfer(to string, amount *big.Int) error {
 	return l.client.ProcessTransaction(tx)
 }
 
-func (l *EthereumLinkToken) TransferAndCall(to string, amount *big.Int, data []byte) error {
+func (l *EthereumLinkToken) TransferAndCall(to string, amount *big.Int, data []byte) (*types.Transaction, error) {
```

Comment:
Nice one! 😊 I think it's much better to have the `TransferAndCall` method have the same return type as the methods belonging to the `LinkToken` struct to avoid confusion.

Reply:
I think this change actually makes the return value different than other functions :sweat_smile:, but other functions haven't needed the tx as output yet.